### PR TITLE
feat(medusa): Support for Maldivian Currency

### PR DESCRIPTION
**Pull request by haabnaseem**
Added Maldivian Currency to the list

### CODE REVIEW

**Comment by haabnaseem** [comment]
@olivermrbl can you please approve, if it's good to go? 
thanks much!

**Comment by olivermrbl** [comment]
@haabnaseem Can I get you to rebase off `develop`? The PR currently holds a lot of changes from `master` since I changed the base branch 💪

File: packages/medusa/src/utils/currencies.ts [hunk 1]
```diff
@@ -675,6 +675,15 @@ export const currencies: Record<string, Currency> = {
     code: "MUR",
     name_plural: "Mauritian rupees",
   },
+  MVR: {
+    symbol: "MVR",
+    name: "Maldivian Rufiyaa",
+    symbol_native: "ރ",
+    decimal_digits: 0,
```
Comment (olivermrbl) [inline packages/medusa/src/utils/currencies.ts:682]: question: Most documentation says it has 2 decimals. Can I get you to check up on that?

Comment (haabnaseem) [reply packages/medusa/src/utils/currencies.ts]: you are right!. updated now

**Review by haabnaseem — COMMENTED** [review]
update to 2 decimal digita

**Review by olivermrbl — CHANGES_REQUESTED** [review]
todo: Fix snap shot in integration tests. See failed pipeline for more info.

**Comment by haabnaseem** [comment]
> todo: Fix snap shot in integration tests. See failed pipeline for more info.

@olivermrbl tests are failing due to "tax_inclusive" flag ? what shall I do now

**Review by olivermrbl — APPROVED** [review]

**Comment by riqwan** [comment]
Hey, thanks for the PR! Since v2 brought a lot of architectural changes, we will be closing this PR since it no longer applies to our new setup.

If you would still like to open this PR, please rebase with the v1 branch and re-request a review. If you would like to open this change for v2 as well, please reopen with the v2 branch as base.

**Pull request closed without merging**